plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryReadClientFactory.java

```diff
@@ -50,13 +53,23 @@ BigQueryReadClient create(ConnectorSession session)
         Optional<Credentials> credentials = credentialsSupplier.getCredentials(session);
 
         try {
+            ChannelPoolSettings channelPoolSettings = ChannelPoolSettings.builder()
+                    .setInitialChannelCount(bigQueryConfig.getRpcInitialChannelCount())
+                    .setMinRpcsPerChannel(bigQueryConfig.getMinRpcPerChannel())
+                    .setMaxRpcsPerChannel(bigQueryConfig.getMaxRpcPerChannel())
+                    .setMinRpcsPerChannel(bigQueryConfig.getRpcMinChannelCount())
+                    .setMaxChannelCount(bigQueryConfig.getRpcMaxChannelCount())
+                    .build();
```

Comment:
How about moving to the constructor? I suppose we don't need to build `ChannelPoolSettings` per `create` method call.